client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -46,6 +46,41 @@ float smoothTriangleWave(float x)
 	return smoothCurve(triangleWave(x)) * 2.0 - 1.0;
 }
 
+#if (MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_TRANSPARENT || MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_OPAQUE || MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_BASIC) && ENABLE_WAVING_WATER
```

Comment:
Too long, this should be shortened.
```suggestion
#if (MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_TRANSPARENT || MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_OPAQUE 
|| MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_BASIC) && ENABLE_WAVING_WATER
```

Reply:
I agree that a line-break is required for this line.

Reply:
But not like this. Terminate preprocessor line breaks using a backslash or this will be interpreted as a regular code line.

Reply:
There are lots of long #IFDEFs in there. I'm not convinced that breaking them up this way makes anything more readable.

Reply:
Not more readable, but follows the code style guidelines.
> Try to keep lines under 80 characters. It's okay if it goes over by a few, but 90 character lines or larger are definitely unacceptable.

https://dev.minetest.net/Code_style_guidelines
PS: This glorious line is 195 characters long, which requires at least two new lines.

Reply:
I agree the long lines should be wrapped at <=90 columns (with tab size 4).

Reply:
Done... Also fixed the other long #IFDEF lines.